### ISSUE TITLE
Spot Check: Our Supporters Page

### ISSUE THREAD

**Issue by mattpen** [phetsims/qa#719]
QA, please complete a quick spot check for the new Our Supporters page. The goal of the test is to make sure the format looks consistent across locales and platforms.  The only interactive element is the expandable text button for "Past Supporters" at the bottom of the page.

Please check at least 3 locales:
- [ ] English - https://ox-dev.colorado.edu/en/our-supporters
- [ ] Non-english LTR locale - e.g.  https://ox-dev.colorado.edu/es/our-supporters
- [ ] RTL locale - e.g. https://ox-dev.colorado.edu/ar_SA/our-supporters

Please check on all supported platforms:
- [x] Latest macOS, Safari (Time = .25 hr)
- [x] Windows 10, Chrome (Time = .25)
- [x] Windows 10, Firefox (Time = .25)
- [x] Latest iOS, Safari (Time = .25 hr )
- [x] Latest ChromeOS, Chrome (Time = .25 hr )
- [x] Windows 11 Chrome (Time = 1.5 hr)

Please check the supported a11y platforms:
- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver 

Please open issues for any problems found in https://github.com/phetsims/website-meteor/issues/new.

This issue is ready for testing whenever there is capacity.  I assigned @kathy-phet and @KatieWoe for priority determination.

**Comment by mattpen** [comment]
Let's do this test after #720.  Placing on hold for now.

**Comment by KatieWoe** [comment]
QA is done

**Comment by mattpen** [comment]
Thanks @KatieWoe
